src/main/scala/scalismo/statisticalmodel/StatisticalVolumeIntensityModel.scala

```diff
@@ -0,0 +1,100 @@
+package scalismo.statisticalmodel
+
+import breeze.linalg.DenseVector
+import scalismo.common._
+import scalismo.geometry._
+import scalismo.mesh._
+import scalismo.utils.Random
+
+import scala.reflect.ClassTag
+import scala.reflect.runtime.universe.TypeTag
+
+trait StatisticalVolumeIntensityModel[S] {
+
+  def referenceMesh: TetrahedralMesh3D
+
+  def referenceMeshField: ScalarVolumeMeshField[S]
+
+  def shape: StatisticalVolumeMeshModel
+
+  def intensity: DiscreteLowRankGaussianProcess[_3D, UnstructuredPointsDomain[_3D], S]
+
+  def landmarks: Map[String, Landmark[_3D]] = Map.empty[String, Landmark[_3D]]
```

Comment:
what is the landmarks for?

Reply:
Landmarks will be removed. See the comment by @marcelluethi above.